src/components/Form/UserForm.vue

```diff
@@ -0,0 +1,109 @@
+<template>
+  <v-text-field
+    prepend-inner-icon="mdi-account"
+    label="First name"
+    required
+    v-model="userData.firstName"
+    :rules="smallFieldRules"
+    :counter="10"
+  ></v-text-field>
+
+  <v-text-field
+    prepend-inner-icon="mdi-account"
+    label="Last name"
+    required
+    v-model="userData.lastName"
+    :rules="smallFieldRules"
+    :counter="10"
+  ></v-text-field>
+
+  <v-text-field
+    prepend-inner-icon="mdi-email-outline"
+    label="Email"
+    required
+    v-model="userData.email"
+    :rules="emailRules"
+    :counter="50"
+  ></v-text-field>
+
+  <v-text-field
+    v-if="!isEditPage"
+    prepend-inner-icon="mdi-lock-outline"
+    :append-inner-icon="isPasswordVisible ? 'mdi-eye-off' : 'mdi-eye'"
+    label="Password"
+    :type="isPasswordVisible ? 'text' : 'password'"
+    required
+    v-model="userData.password"
+    @click:append-inner="isPasswordVisible = !isPasswordVisible"
+    :rules="passwordRules"
+  ></v-text-field>
+
+  <v-text-field
+    prepend-inner-icon="mdi-map-marker"
+    label="Address"
+    v-model="userData.address"
+    :counter="50"
+  ></v-text-field>
+
+  <v-text-field
+    prepend-inner-icon="mdi-cellphone"
+    label="Phone"
+    v-model="userData.phone"
+  ></v-text-field>
+
+  <v-text-field
+    prepend-inner-icon="mdi-cellphone"
+    label="Phone2"
+    v-model="userData.phone2"
+  ></v-text-field>
+
+  <v-text-field
+    prepend-inner-icon="mdi-calendar"
+    label="Birth date"
+    v-model="userData.birthDate"
+    type="date"
+    :rules="inputRules"
+  ></v-text-field>
```

Comment:
this is the problematic date that you mentioned, right? I read a little and there is no pretty way of dealing with this. Vuetify really doesn't have a good wrapper around this and the native text-field with type format understands only the yyyy-mm-dd. There is an alternative to try: `type="datetime-local"`. I don't like this, because we are introducing time as well this way but maybe it's more flexible with formats?

Reply:
If it doesn't work, leave it as it is

Reply:
yes, thats why i did 2 methods so that I can work around with the date coming from/to the input field and we dont get the warning. tried datetime-local, its just extends and give more information, nothing more